src/cms/views/offers/offer_actions.py

```diff
@@ -37,8 +37,7 @@ def deactivate(request, region_slug, offer_template_slug):
     offer.delete()
     messages.success(
         request,
-        _('Offer "%(offer_name)s" was successfully deactivated.')
-        % {"offer_name": template.name},
+        _("Offer was successfully deactivated.") % {"offer_name": template.name},
```

Comment:
I think our reason is, that we generally do not name the objects we save/change/etc. However, I do not have a strong opinion here. If this is a case were it makes more sense than usual (i.e. saving pages), then lets put the name back in again.

Reply:
Since the offer list is very different from our other content types (e.g. we don't have separate forms and the activate-buttons are next to each other), I'd just include it here.

Reply:
Allright then, I'll change it.